conformance/Makefile.am

```diff
@@ -2,7 +2,11 @@
 
 conformance_protoc_inputs =                                    \
   conformance.proto                                            \
-  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto
+  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto 
+  
+conformance_proto2_protoc_inputs =                             \
```

Comment:
why do you need a second set of input and to invoke ptotoc twice in all the generation cases?  You should be able to generate proto2 and proto3 syntax files all at once.  Since you're compiling them into a single binary, I'm assuming they have unique proto package/name combinations, and that isn't the problem.